component/azstorage/block_blob.go

```diff
@@ -582,11 +583,19 @@ func (bb *BlockBlob) List(prefix string, marker *string, count int32) ([]*intern
 				return blobList, nil, err
 			}
 		} else {
+			var mode os.FileMode = 0
+			if blobInfo.Properties.Permissions != nil {
+				mode, err = getFileMode(*blobInfo.Properties.Permissions)
+				if err != nil {
+					log.Err("BlockBlob::List : Failed to get file mode for %s [%s]", *blobInfo.Name, err.Error())
+					return blobList, nil, err
```

Comment:
Should we fail the list call here or just set the mode to 0 and move to the next blob?

Reply:
Maybe we can move on but atleast we should log that we failed to get file mode so its easier to debug